package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "nfdiv-case-api",
+  "version": "1.0.0",
+  "description": "Utility to help creating CCD config definitions",
+  "main": "index.js",
+  "repository": "git@github.com:hmcts/nfdiv-case-api.git",
+  "license": "MIT",
+  "private": true,
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "config": {
+    "local": {
+      "cosUrl": "http://nfdiv-cos:4012",
```

Comment:
do we cos now ? callbacks will be handled by this case api library

Reply:
nope, should be nfdiv-case-api

Reply:
I have removed these completely as we can do this in Java rather than when the spreadsheet is created